ts/client/Components/Dumb/SVG.ts

```diff
@@ -2,7 +2,7 @@
 
 import * as React from 'react';
 import { findDOMNode, unmountComponentAtNode } from 'react-dom';
-import * as SVGInjector from 'svg-injector';
+import SVGInjector = require('svg-injector');
```

Comment:
This is TS related. I struggled to make it work without TS errors. This is the pattern i found, if you have better options, let me know about it.